src/ipelib/flip/CTIPE-int.f

```diff
@@ -565,14 +565,24 @@ SUBROUTINE CTIPINT(
       sza_south = sza(jmax - 4) * 180.0 / 3.14159
 
       IF(sza_north.LT.103.0.OR.sza_south.LT.103.0) then
+! 
+! flux tube 158 has an apex height of 127.6
+! that's the last one we call photoelectrons on...
+!
+! GHGM - had an issue at 158 - trying 156 as the max

```

Comment:
comment here doesn't match with the if statement below, just clean up to whatever is correct